doc/Language/traps.pod6

```diff
@@ -921,46 +921,64 @@ of the assignment operators instead:
 
 =head1 Regexes
 
-=head2 C«<{$x}>» vs C«$($x)»: Implicit EVAL
-
-Sometimes you may need to match a generated string in a regex. This can be done
-using C<$(…)> or C«<{…}>» syntax:
-
-=for code
-my $x = ‘ailemac’;
-say ‘I ♥ camelia’ ~~ / $($x.flip) /; # OUTPUT: «｢camelia｣␤»
-say ‘I ♥ camelia’ ~~ / <{$x.flip}> /; # OUTPUT: «｢camelia｣␤»
-
-However, the latter only works I<sometimes>.
-
-Internally C«<{…}>» EVAL-s the given string inside an anonymous regex, while
-C<$(…)> lexically interpolates the given string. So C«<{…}>» immediately breaks
-with more complicated inputs. For example:
-
-=for code
-my $x = ‘ailemac#’;
-say ‘I ♥ #camelia’ ~~ / $($x.flip) /; # OUTPUT: «｢#camelia｣␤»
-#                    ⚠ ↓↓ WRONG ↓↓ ⚠
-say ‘I ♥ #camelia’ ~~ / <{$x.flip}> /;
-# OUTPUT:
-# ===SORRY!===
-# Regex not terminated.
-# at EVAL_0:1
-# ------> anon regex { #camelia}⏏<EOL>
-# Malformed regex
-# at EVAL_0:1
-# ------> anon regex { #camelia}⏏<EOL>
-#     expecting any of:
-#         infix stopper
-
-Therefore, try not to use C«<{}>» unless you really need EVAL.
-
-Note that even though EVAL is normally considered unsafe, in this case
-it is restricted to a set of safe operations (which is why it works
-without L<MONKEY-SEE-NO-EVAL|
-/language/pragmas#index-entry-MONKEY-SEE-NO-EVAL-MONKEY-SEE-NO-EVAL> pragma).
-In theory, careless use of C«<{}>» will only result in an exception being
-thrown, and should not introduce security issues.
+=head1 Interpolation constructs
```

Comment:
Maybe this should be `=head2`? Also, it might be better if it was formulated like an actual trap, something like "Problems in the interpretation of regex interpolation constructs"

Reply:
`head1` was my bad. - As to the title, I picked smth short because the long titles wrap somewhat clumsily when rendered in HTML. But if we go for long, how about 'Interpolation constructs that behave similarly, some of the time' ?

Reply:
OK